systemd/docker-novolume-plugin.socket

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Docker novolume plugin Socket for the API
+#PartOf=docker-novolume-plugin.service
+
+[Socket]
+ListenStream=/run/docker/plugins/docker-novolume-plugin.sock
+SocketMode=0660
+SocketUser=root
+SocketGroup=docker
```

Comment:
we're not using the docker group, so I think we should set group to `root`, or maybe just remove both SocketUser and SocketGroup, which will default to root. @rhatdan wdyt?